File: app/src/main/java/com/example/user/eventest/MainView.java
package com.example.user.eventest;

import com.example.user.eventest.model.Memo;

/**
 * Created by DR
 * on 26.02.2018.
 */

public interface MainView {
    boolean hasFocusNote();

    void setEditViewsGone();

    void setEditViewsVisible();

    Memo getEditedMemo();

    void setEditedMemo(Memo memo);

    void getSelectedMemoToEdit(int position);
}

File: app/src/main/java/com/example/user/eventest/MainView.kt
package com.example.user.eventest

import com.example.user.eventest.model.Memo

/**
 * Created by DR
 * on 26.02.2018.
 */

interface MainView {
    fun hasFocusNote(): Boolean
    fun setEditViewsGone()
    fun setEditViewsVisible()
    fun getEditedMemo(): Memo
    fun setEditedMemo(memo: Memo)
    fun getSelectedMemoToEdit(position: Int)
    fun updateWidget()
}
